qa/workunits/cephtool/test.sh

```diff
@@ -351,14 +351,16 @@ function test_tiering_1()
   expect_false ceph osd tier cache-mode cache forward
   expect_false ceph osd tier cache-mode cache forward --yes-i-really-mean-it
   expect_false ceph osd tier cache-mode cache readonly
-  ceph osd tier cache-mode cache proxy
+  # proxy is removed/deprecated
+  expect_false ceph osd tier cache-mode cache proxy
```

Comment:
could you drop this line? no need to test an unsupported cache mode.

Reply:
this line is testing that `ceph osd tier cache-mode cache proxy` really fails as expected, similar to the previous three lines which test the same for `forward` and `readonly` caching modes.